content/articles/pse-june-2025-newsletter.md

```diff
@@ -0,0 +1,90 @@
+---
+authors: ["PSE Comms Team"]
+title: "PSE June 2025 newsletter"
+image: "/articles/pse-june-2025/cover.webp"
+tldr: "Check out what PSE teams have been focused on in June 2025!"
+date: "2025-06-13"
+tags:
+  [
+    "newsletter",
+    "post quantum cryptography",
+    "private proof delegation",
+    "client side proving",
+    "machina io",
+    "voprf",
+    "tlsnotary",
+    "maci",
+    "zk-kit",
+    "mpc framework",
+    "mopro",
+    "semaphore",
+  ]
+projects:
+  [
+    "post-quantum-cryptography",
+    "private-proof-delegation",
+    "client-side-proving",
+    "machina-io",
+    "voprf",
+    "tlsn",
+    "maci",
+    "zk-kit",
+    "mpc-framework",
+    "mopro",
+    "semaphore",
+  ]
+---
+
+Here’s a round-up of what PSE teams have been up to in June!
+
+## 🧪 Research Highlights
+
+### [Client-Side Proving](https://pse.dev/en/projects/client-side-proving)
```

Comment:
@kalidiagne how should links be created? 
A) relative (e.g. `/projects/client-side-proving`)
B) absolute (e.g. `https://pse.dev/en/projects/client-side-proving`)
C) something else

I would guess relative is what we should do?

Could you please add a reference for that on the README? 
https://github.com/privacy-scaling-explorations/pse.dev/blob/main/content/articles/README.md

Thanks!

Reply:
Sure! let me add this into the docs!